framework/accessibility/internal/accessibilitycontroller.cpp

```
@@ -286,12 +286,22 @@ void AccessibilityController::announce(const QString& announcement)
     }
 
     const Item& focused = findItem(m_lastFocused);
     if (!focused.isValid()) {
+        // With no focused item there is nothing to revoice, so fall back to a
+        // plain announcement event addressed to the main window.
+        QWindow* window = mainWindow()->qWindow();
+        if (!window) {
+            return;
+        }
+
+        QAccessibleAnnouncementEvent event(window, announcement);
```

Clang (14.0.6)

[warning] variable 'event' is not initialized

(cppcoreguidelines-init-variables)